frontend/test/metabase/admin/tasks/containers/Logs.unit.spec.js

```diff
@@ -6,6 +6,16 @@ import { UtilApi } from "metabase/services";
 
 describe("Logs", () => {
   describe("log fetching", () => {
+    beforeEach(() => {
+      const xhrMockClass = () => ({
+        open: jest.fn(),
+        send: jest.fn(),
+        setRequestHeader: jest.fn(),
+      });
+
+      window.XMLHttpRequest = jest.fn().mockImplementation(xhrMockClass);
+    });
+
```

Comment:
Should we maybe use [`xhr-mock`](https://www.npmjs.com/package/xhr-mock) here? It's already in use for other unit-tests ([example](https://github.com/metabase/metabase/blob/3c71c5186dd4c9003b756f4db25498968283e0b3/frontend/test/metabase/entities/databases.unit.spec.js#L11))

Reply:
Good catch, thanks @kulyk! I've made the tweaks.